RileyLinkKit/PumpOpsSynchronous.swift

```diff
@@ -464,8 +464,17 @@ class PumpOpsSynchronous {
                     throw error
                 }
             }
-            
-            NSLog("Fetched page %d: %@", pageNum, pageData as NSData)
+            if true {
+                let logString = NSString(format: "Fetched page %d: %@", pageNum, pageData as NSData ) 
+                let sz = logString.length
```

Comment:
I like that.

(my character number was based on a space after every 4 bytes `1024 * 2 + (1024 / 4)`)